bin/standalone-installer-unix

```diff
@@ -92,7 +92,7 @@ Nextstrain CLI ($installed_version) installed to $DESTINATION.
 To make the "nextstrain" command available in your default shell ($shell)
 without using the full path, please run these two commands now:
 
-    echo 'eval "\$("$DESTINATION/nextstrain" init-shell $shell)"' >> $rc
+    echo '\neval "\$("$DESTINATION/nextstrain" init-shell $shell)"' >> $rc
```

Comment:
bash (and many other shells) won't expand `\n` within this string (either at expansion time or within `echo`).

I do see that zsh will (at least by default).

We can use `echo -e` instead as long as we ensure we don't use any other recognized escapes (we currently don't).

Or we can switch to something that doesn't mix an interpreted string with uninterpreted data, like

```
printf '\n%s\n' 'eval "\$("$DESTINATION/nextstrain" init-shell $shell)"' >> $rc
```

Another alternative is using the syntax for an interpreted string literal and string concat:

    echo $'\n''eval "\$("$DESTINATION/nextstrain" init-shell $shell)"'

And if we do that, for consistency across shells we can also throw in `-E` to avoid `echo` interpreting the string.

Reply:
> Use printf to avoid expanding '\n' in bash, zsh, and hopefully all other
shells.

Ah, not to avoid expanding `\n` but to consistently expand it.

Reply:
Oops, I got mixed up. Fixed in ac5d20e21c3ed2ca3cd23cc207f164a10ff73efb